example/wallet_example.go

```diff
@@ -0,0 +1,76 @@
+// Copyright (C) 2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+package main
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/utils"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/network"
+
+	"github.com/ava-labs/avalanche-tooling-sdk-go/wallet/p-chain/txs"
+)
+
+func CreateSubnet() error {
+	ctx, cancel := utils.GetTimedContext(120 * time.Second)
+	defer cancel()
+	network := network.FujiNetwork()
+
+	localWallet, err := wallet.NewLocalWallet()
+	if err != nil {
+		return fmt.Errorf("failed to create wallet: %w", err)
+	}
+
+	existingAccount, err := localWallet.ImportAccount(ctx, "/Users/raymondsukanto/.avalanche-cli/key/newTestKey.pk")
```

Comment:
Can we create a pk file somewhere in this folder and import it using a relative path?